Misc/NEWS.d/next/Core and Builtins/2018-05-13-01-26-18.bpo-33475.rI0y1U.rst

```diff
@@ -0,0 +1,2 @@
+Fixed miscellaneous bugs in onverting annotations to strings and optimize
```

Comment:
converting, optimized (to make tense consistent), parentheses (on the next line)